rocketmq-docker/README.md

```diff
@@ -1,51 +1,47 @@
 # Apache RocketMQ Docker module
 
-Apache RocketMQ Docker module provides Dockerfile and bash scripts for building docker image.
+Apache RocketMQ Docker module provides Dockerfiles and scripts for RocketMQ.
 
+This repository includes the following: 
 
-cd 4.0.0-incubating/broker/ or cd 4.0.0-incubating/namesrv/ 
+1. Dockerfile and scripts for RocketMQ base image;
+2. Dockerfile and scripts for RocketMQ run in following 3 scenarios:
+- RocketMQ runs on single Docker daemon;
+- RocketMQ runs with docker-compose;
+- RocketMQ runs on Kubernetes.
 
-## Prequirement
+## Quick start: Build and run RocketMQ with a single instance
 
-* Docker Engine
-* Linux / windows 
+### For Docker
 
-## Build:
+Run: 
 
-### For Linux
-```bash
-sh docker_build.sh
-```
-### For Windows
-```
-.\docker_build.cmd
 ```
+cd 4.2.0
 
-## Run:
+./play-docker.sh
 
-### For Linux
-```
-sh docker_run.sh
-```
-### For Windows
-```
-.\docker_run.cmd
 ```
 
-## Play:
+### For docker-compose
 
-Build broker and namesrv images, then run them at local. The name server address of broker is automatically configured.
+Run:
 
 ```
-cd 4.0.0-incubating
-```
+cd 4.2.0
```

Comment:
@XiaoZYang Partially agree. Currently I'm not sure if the original running scripts from apache/rocketmq upstream will always keep the same in upcoming new releases. I'm also considering if we need to submit a PR to project apache/rocketmq to adjust and solid the running scripts so that we don't need to care this in our Dockerfile. Anyway, I believe that we can enhance this part later once the scripts become stable.

Reply:
@huanwei Submitting a PR to project `apache/rocketmq` is great which gets rid of building custom scripts for docker.

Reply:
@huanwei @XiaoZYang LGTM, we could submit another pr to polish here.